### PR TITLE
[release/8.0][browser] drop old npm dependencies

### PR DESCRIPTION
**Pull request by pavelsavara**
Partial backport of https://github.com/dotnet/runtime/pull/103639 to release/8.0

 - drop samples with old NPM dependencies
 - fix test script to allow loading latest `ws` module in NodeJS

## Customer Impact

None, this is just to pacify security scanner and only touches non-production code.

## Testing

Automated CI

## Risk
Low risk.

### CODE REVIEW

**Review by ilonatommy — APPROVED** [review]

**Comment by pavelsavara** [comment]
@carlossanlop do we need tactics approval for this ?

**Comment by steveisok** [comment]
> @carlossanlop do we need tactics approval for this ?

No, for tests and infra it's tell-mode. Just float an email after you merge.

**Comment by carlossanlop** [comment]
@pavelsavara friendly reminder that code complete for the August Release is July 15th. If you want this fix to be included in that release, please merge this PR before that date.

If this is a tell-mode / test-only change, we can directly apply the `servicing-approved` label.

**Pull request merged**
